includes/bootstrap/SWSFeatureContext.php

```diff
@@ -696,4 +695,22 @@ public function iShouldSeeNoDuplicateHtmlWith($arg1)
       throw new ExpectationException($message, $session);
     }
   }
+
+  /**
+   * @Given I am logged out
+   */
+  public function iAmLoggedOut()
+  {
+    throw new PendingException();
```

Comment:
Why are why changing this function to throw an error rather than leaving it to log a user out?